pod/video/transcript.py

```diff
@@ -24,29 +30,51 @@
     FILEPICKER = False
     from pod.main.models import CustomFileModel
 
-FFMPEG = getattr(settings, 'FFMPEG', 'ffmpeg')
-FFMPEG_NB_THREADS = getattr(settings, 'FFMPEG_NB_THREADS', 0)
+AUDIO_SPLIT_TIME = getattr(settings, 'AUDIO_SPLIT_TIME', 600)  # MOVE TO 600
+# time in sec for phrase length
+SENTENCE_MAX_LENGTH = getattr(settings, 'SENTENCE_MAX_LENGTH', 3)
 
-DS_PARAM = getattr(settings, 'DS_PARAM', dict())
+DESIRED_SAMPLE_RATE = getattr(settings, 'DESIRED_SAMPLE_RATE', 16000)
```

Comment:
With latest alpha, you can query that from the model: https://deepspeech.readthedocs.io/en/latest/Python-API.html#native_client.python.Model.sampleRate

Reply:
ok, I'll modify it

Reply:
Done